.circleci/config.yml

```diff
@@ -1139,21 +1103,34 @@ workflows:
               agent-instance-type: ["p2.8xlarge"]
               max-dynamic-agents: [8]
 
-      - parallel-tests:
+      - test-e2e-aws:
+          name: test-e2e-gpu-parallel
           context: aws
           filters:
             branches:
               only: master
           requires:
             - package-and-push-system-dev
+          matrix:
+            parameters:
+              parallelism: [2]
+              agent-instance-type: ["p2.8xlarge"]
+              mark: ["parallel"]
+              slack-mentions: ["${SLACK_USER_ID}"]
 
-      - e2e-gpu-tests:
+      - test-e2e-aws:
+          name: test-e2e-gpu-single
           context: aws
           filters:
             branches:
               only: master
           requires:
             - package-and-push-system-dev
+          matrix:
+            parameters:
+              parallelism: [2]
+              mark: ["e2e_gpu"]
+              slack-mentions: ["${SLACK_USER_ID}"]
```

Comment:
blocking: I just realized this is not going to work for the slack mentions.. This environment variable needs to be set by the job that sends the slack notification

Reply:
this is what that `set-slack-user-id` command does

Reply:
I set it in https://github.com/determined-ai/determined/pull/852/files#diff-1d37e48f9ceff6d8030570cd36286a61R941. Does this not work?

Reply:
ohh.. interesting that should actually work 💯

Reply:
a little tricky but i like it :)